packages/expo-screen-orientation/android/src/main/java/expo/modules/screenorientation/ScreenOrientationModule.kt

```diff
@@ -32,6 +32,8 @@ class ScreenOrientationModule : Module(), LifecycleEventListener {
   override fun definition() = ModuleDefinition {
     Name("ExpoScreenOrientation")
 
+    Events("expoDidUpdateDimensions")
```

Comment:
Might be worth adding a comment that this event is actually not used from native, but suppresses warnings from NativeEventEmitter. I hope we'll soon make event emitters built-in for module objects and then we could remove it from the definition.